src/accessories/simplicam.js

```diff
@@ -86,6 +92,33 @@ class SS3SimpliCam {
         }
     }
 
+    startListening() {
+        this.log('Camera listening to alarm events...');
+        this.simplisafe.subscribeToEvents((event, data) => {
+            this.log(`Camera received new event from alarm: ${event}`);
+            if (this.Service && data.sensorName == this.name) {
```

Comment:
there shouldn't be a need for `this.Service` in the if statement

Reply:
I think that came from your startListening method :) https://github.com/nzapponi/homebridge-simplisafe3/blob/e5675255e6b91e90d162fd1b9239a122b11dad8c/src/accessories/alarm.js#L145